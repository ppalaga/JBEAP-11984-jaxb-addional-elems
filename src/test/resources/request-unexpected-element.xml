<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:demo="http://jbeap11984.issues.wildfly.org/">
  <soap:Header />
  <soap:Body>
    <demo:hello>
      <arg0>Frank</arg0>
      <arg1>100</arg1>
      <unexpected>dummy</unexpected>
    </demo:hello>
  </soap:Body>
</soap:Envelope>