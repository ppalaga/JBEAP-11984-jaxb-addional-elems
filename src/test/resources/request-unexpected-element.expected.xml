<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><ns2:helloResponse xmlns:ns2="http://jbeap11984.issues.wildfly.org/"><return>Hello Frank 100 times</return></ns2:helloResponse></soap:Body></soap:Envelope>